the-blue-alliance-ios.xcodeproj/project.pbxproj

```diff
@@ -481,6 +484,7 @@
 		92D855DA1EC6B8F100572F27 /* Event */ = {
 			isa = PBXGroup;
 			children = (
+				303F82491EF593BE007CADA1 /* EventRankingsTableViewController.swift */,
```

Comment:
Super picky again but - can we just move this down below the InfoTableViewController in the file hierarchy